### DISCUSSION TITLE
Bold, inline code and inline math working weird.

### DISCUSSION THREAD

**Discussion by Wmbat** [nvim-neorg/neorg#167] [Q&A] [answered]
hey, 

I've been writing some notes in neorg and i've found some odd behaviour and I couldn't find any video regarding what the correct behaviour for those should be. This is what words in bold, inline code and other specifiers look https://imgur.com/8lr0R6t. 

I'm using the unstable branch and i've loaded the concealer as default. is the presence of the bold delimiters normal?

**Comment by vhyrro** [comment, marked as answer]
Hi!

Yeah, their presence is normal... kind of. By default they will be shown because of your conceal level. You want to set this to something higher than 0 (preferably 2, see `:h conceallevel` for more info on that).

In vimscript:
```vim
set conceallevel=2
```

In lua:
```lua
vim.opt.conceallevel = 2
```

Related topic: <https://github.com/nvim-neorg/neorg/discussions/135#discussioncomment-1412130>.
When it comes to maths no special concealing will be done for that (no highlighting will be applied to it either just yet). I'm working on that in our treesitter implementation to support it.

Hope that helps! :)

**Reply by Wmbat** [reply]
Ok, thanks a lot. This fixed all the bold, underline and inline code, but the inline math and the math `@math` code block still don't work. I have everything up to date, has the math related concealers not been added yet?

**Reply by vhyrro** [reply]
The inline math shouldn't work as I said earlier but the `@math` code block should highlight latex just fine 🤔 

Do you have the latex treesitter parser installed? It's required to inject the latex parser into the `@math` content :)

**Reply by Wmbat** [reply]
I do, I also noticed the `@table` don't work either. I must have a setting disabled somewhere
